ladybug_vtk/model.py

```diff
@@ -0,0 +1,88 @@
+"""A VTK representation of HBModel."""
```

Comment:
We should probably keep this module in honeybee-vtk. In the ladybug world, there is no such concept as a model.

Reply:
I saw how you are using it in your test example. We should probably update the description.